.travis.yml

```diff
@@ -27,10 +24,15 @@ before_script:
   - chmod 777 tests/phpunit/unit/Nut/resources/
   - mkdir -p app/cache
   - chmod 777 app/cache
+  - if [ $TRAVIS_PHP_VERSION = '5.6' ]; then PHPUNIT_FLAGS="--coverage-clover coverage.clover"; fi
+
+script:
+  - phpunit $PHPUNIT_FLAGS
+  - ./run-functional-tests
 
 after_script:
-  - ./vendor/bin/ocular code-coverage:upload --format=php-clover coverage.xml
-  
+  - if [ $TRAVIS_PHP_VERSION = '5.6' ]; then wget https://scrutinizer-ci.com/ocular.phar && php ocular.phar code-coverage:upload --format=php-clover coverage.clover; fi
```

Comment:
Yes it can be, it's only needed along with the code coverage report which we've dropped so not relevant any more.